nixos/modules/services/web-apps/nexus.nix

```diff
@@ -19,6 +18,16 @@ in
         description = "Package which runs Nexus3";
       };
 
+      jdkPackage = mkOption {
```

Comment:
@Janik-Haag when merging old PRs, please make sure they are up to standard. This option triggers the following warning ``trace: warning: literalExample is deprecated, use literalExpression instead, or use literalMD for a non-Nix description.`` and should use mkPackageOption.

Reply:
#244323

Reply:
Sorry, I missed that, thank you for pointing this out.